dist/spec.yaml

```diff
@@ -2058,7 +2058,7 @@ paths:
         '200':
           description: Success
           schema:
-            $ref: '#/definitions/DomainListResponse'
+            $ref: '#/definitions/Domain'
```

Comment:
This response is neither a Domain, nor a list of Domains - https://developer.okta.com/docs/reference/api/domains/#domainlistresponse-example

Reply:
True, let me fix it quickly.

Reply:
I mean, format of response is different, looks like list_domains response doesn't consist of Domain objects. First difference: key "domains", second - list of some objects which are only similar to Domain.
We can try do something like:
```js
schema:
   domains:
       $ref: '#/definitions/Domain'
```
But I'm not sure if it will work